src/components/Home/ConfigFileDropZone/ConfigFileDropZone.tsx

```diff
@@ -27,21 +28,13 @@ export const ConfigFileDropZone: FunctionComponent<ConfigFileDropZone> = ({ onCo
   };
   const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
 
-  const dropZoneCSS =
-    errorMessage || error
-      ? `border-dashed border-2 items-center border-red flex flex-col pt-16 pb-16 px-4 text-center ${
-          isDragActive ? "bg-gray-300" : "bg-red-100"
-        }`
-      : `border-dashed border-2 items-center border-gray-300 flex flex-col pt-16 pb-16 px-4 text-center ${
-          isDragActive ? "bg-gray-300" : "bg-white"
-        }`;
-
   return (
     <>
       <Title className="mb-8">Create Document</Title>
       <div {...getRootProps()}>
         <input data-testid="config-file-drop-zone" {...getInputProps()} />
-        <div className={dropZoneCSS}>
+        <DropZone isDragActive={isDragActive} error={Boolean(errorMessage || error)}>
```

Comment:
```suggestion
        <DropZone isDragActive={isDragActive} error={!!(errorMessage || error)}>
```

Reply:
https://gist.github.com/arthurvi/66cb1e2bcfc92f99f465e0db04264367 
i like to use boolean cause more explicit